x-pack/plugins/spaces/public/copy_saved_objects_to_space/copy_saved_objects_to_space_action.tsx

```diff
@@ -5,25 +5,43 @@
  * 2.0.
  */
 
-import { EuiLoadingSpinner } from '@elastic/eui';
-import React from 'react';
+import React, { lazy, useEffect, useState } from 'react';
 
 import { i18n } from '@kbn/i18n';
+import type { NotificationsStart, StartServicesAccessor } from 'src/core/public';
 import type { SavedObjectsManagementRecord } from 'src/plugins/saved_objects_management/public';
 
 import { SavedObjectsManagementAction } from '../../../../../src/plugins/saved_objects_management/public';
+import type { PluginsStart } from '../plugin';
+import { SuspenseErrorBoundary } from '../suspense_error_boundary';
 import type { CopyToSpaceFlyoutProps } from './components';
 import { getCopyToSpaceFlyoutComponent } from './components';
 
-const Wrapper = (props: CopyToSpaceFlyoutProps) => {
-  const LazyComponent = React.lazy(() =>
-    getCopyToSpaceFlyoutComponent().then((component) => ({ default: component }))
-  );
+const LazyCopyToSpaceFlyout = lazy(() =>
+  getCopyToSpaceFlyoutComponent().then((component) => ({ default: component }))
+);
+
+interface WrapperProps {
+  getStartServices: StartServicesAccessor<PluginsStart>;
+  props: CopyToSpaceFlyoutProps;
+}
+
+const Wrapper = ({ getStartServices, props }: WrapperProps) => {
+  const [notifications, setNotifications] = useState<NotificationsStart | undefined>();
+  useEffect(() => {
+    getStartServices().then(([coreStart]) => {
+      setNotifications(coreStart.notifications);
+    });
+  });
```

Comment:
This side effect will be executed on every render. Is that intentional or did you mean to add an empty dependency array?

Reply:
I'll fix as you suggested in https://github.com/elastic/kibana/pull/91976#discussion_r583107336